conf/messages.en

```diff
@@ -877,6 +883,13 @@ addSecondOtherOfficials.p3 = Adding information about other officials takes arou
 addedSecondOtherOfficial.checkYourAnswers.title = Check the second other official''s details
 addedSecondOtherOfficial.checkYourAnswers.heading = Check the second other official''s details
 
+# Charity's Other Officials Details Summary Page Messages
+# ----------------------------------------------------------
+otherOfficialsSummary.checkYourAnswers.title = Check the charity''s other official details
```

Comment:
Page title is different, should be from ticket "Check your charity’s other officials details", please check it

Reply:
It was mentioned that we should follow tickets and then will get additional ticket for content changes